### PR TITLE
Remove dependency on isomorphic-fetch

### LINKED ISSUE #1613: [deps] Help with upgrading eslint-plugin-import
@dependabot tried to upgrade https://github.com/jaegertracing/jaeger-ui/pull/1609 but it broke the CI, probably requires some minor code changes to satisfy the new linter version

### PR DESCRIPTION
**Pull request by Yushmanth-reddy**
## Which problem is this PR solving?
- Helps with #1613 

## Short description of the changes
- remove dependency on `isomorphic-fetch`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,14 +18,16 @@
...
     "jsdom": "22.1.0",
     "lerna": "7.1.3",
     "npm-run-all": "4.1.5",
     "prettier": "3.0.0",
     "rxjs-compat": "6.6.7",
     "typescript": "5.1.6"
   },
+  "dependencies": {
+    "isomorphic-fetch": "3.0.0"
+  },
```
Comment (anshgoyalevil) [inline package.json:30]: Was that needed to be moved from ```devDependencies``` to ```dependencies```?

Comment (Yushmanth-reddy) [reply package.json:30]: after changing [scripts/get-changelog.js](https://github.com/jaegertracing/jaeger-ui/pull/1616/files#diff-106ea5da4aa3b89f6740639ff95dc458220eed23cde798e70c95571e8a30e8e7) there was an error ('isomorphic-fetch' should be listed in the project's dependencies, not devDependencies). So I modified it and it worked fine.

File: scripts/get-changelog.js [hunk 2]
```diff
@@ -17,11 +17,11 @@
 // This code will generate changelog entries
 
 const { readFile } = require('fs');
+const { promisify } = require('util');
 // eslint-disable-next-line import/no-extraneous-dependencies
 const fetch = require('isomorphic-fetch');
```
Comment (yurishkuro) [inline scripts/get-changelog.js:22]: Can we remove this dep altogether? It's a polyfill for browsers, but we're using this script only with node.js

Comment (Yushmanth-reddy) [reply scripts/get-changelog.js:22]: Sure, thanks @anshgoyalevil

**Comment by yurishkuro** [comment]
It's still not removed

**Comment by Yushmanth-reddy** [comment]
Don't know why the unit test is failed but the command "yarn install --frozen-lockfile" is running successfully in my local setup.

**Comment by Yushmanth-reddy** [comment]
@yurishkuro Should I make any changes?

**Pull request merged**